app/data_migrations/populate_geography.py

```diff
@@ -5,9 +5,12 @@
 from sqlalchemy.orm import Session
 
 from app.db.models.law_policy import Geography
+from app.db.models.law_policy.geography import GeoStatistics
 
 from .utils import has_rows, load_tree
 
+CPR_DEFINED_GEOS = {"XXA": "No Geography", "XXB": "International"}
```

Comment:
```suggestion
CPR_DEFINED_GEOS = {"XAA": "No Geography", "XAB": "International"}
```

Reply:
Let me make consts so we can use them in the code